<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionConfig</class>
 <widget class="QWidget" name="SessionConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>479</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labPathDSI">
          <property name="toolTip">
           <string>Full path to DragonScript interpreter (dsi), leave empty for default</string>
          </property>
          <property name="text">
           <string>DragonScript Interpreter</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="fraPathDSI">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
             <item>
              <widget class="QLineEdit" name="editPathDSI">
               <property name="toolTip">
                <string>Full path to DragonScript interpreter (dsi), leave empty for default</string>
               </property>
               <property name="placeholderText">
                <string>Full path to DragonScript interpreter (dsi), leave empty for default</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnPathDSI">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labDragengine">
          <property name="toolTip">
           <string>Path to Drag[en]gine Module (&quot;dragonscript&quot; directory)</string>
          </property>
          <property name="text">
           <string>Drag[en]gine Module</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QFrame" name="fraDragengine">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
             <item>
              <widget class="QLineEdit" name="editPathDragengine">
               <property name="toolTip">
                <string>Path to Drag[en]gine Module (&quot;dragonscript&quot; directory)</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Path to Drag[en]gine Module (&quot;dragonscript&quot; directory)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnPathDragengine">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
